Example/Podfile

```diff
@@ -14,7 +14,7 @@ target 'Emission' do
   pod 'Emission', :path => '../'
 
   # As this runs dev, we need the developer web socket
-  pod 'React', :path => react_path, :subspecs => %w(DevSupport BatchedBridge RCTAnimation)
+  pod 'React', :path => react_path, :subspecs => %w(DevSupport RCTAnimation)
```

Comment:
@sarahscott This had to move to the podspec so that we could lint and push the spec, it also means that Eigen doesn’t need to add this detail itself but just rely on `Emission.podspec` to do it all.